bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/ClangdPreferredOptions.java

```diff
@@ -74,7 +74,10 @@ public String queryDriver() {
 
 	@Override
 	public List<String> additionalOptions() {
-		return Arrays.asList(stringValue(metadata.additionalOptions()).split("\\R")); //$NON-NLS-1$
+		var options = stringValue(metadata.additionalOptions());
+		if (options.isBlank())
```

Comment:
Having curly brackets here could improve maintainability. Not sure if this is specified anywhere in the CDT code style.